src/request_collector/server.py

```diff
@@ -49,10 +49,14 @@ def stop(self) -> None:
         self.matrix_listener.join()
 
     def handle_message(self, message: Message) -> None:
-        if isinstance(message, RequestMonitoring):
-            self.on_monitor_request(message)
-        else:
-            log.debug("Ignoring message", message=message)
+        try:
+            if isinstance(message, RequestMonitoring):
+                self.on_monitor_request(message)
+            else:
+                log.debug("Ignoring message", message=message)
+        # add more advanced exception catching
```

Comment:
What "more advanced exception catching" do you expect, here?

Reply:
Maybe something more differentiated with better logs to distinguish. does that make sense?

Reply:
I see this except as a safety measure in case we fail to properly handle some case at a different place. I don't see what I would add here, since other errors are usually better handled closer to where they are thrown.

Reply:
ok then I remove the comment